src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -148,6 +149,9 @@ def main():
     df = _parse_from_metadata_file(load_files, SELECTED_FEATURES)
     docs = _df_to_docs(df, kbase_collection, load_version)
 
+    docs, meta_doc = process_columnar_meta(docs, kbase_collection, load_version, allow_missing_metadata=False)
+
+    # TODO save columnar_attri_meta to import_files/NONE folder
```

Comment:
Does this need to be done then...?

Reply:
I was hoping to get this done with https://kbase-jira.atlassian.net/browse/RE2022-245
so I don't mix a lot of unrelated changes to this PR.

Reply:
: